implementations/micrometer-registry-otlp/build.gradle

```diff
@@ -13,6 +13,10 @@ dependencies {
     testImplementation 'org.awaitility:awaitility'
 }
 
+dockerTest {
+    systemProperty 'otel-collector-image.version', '0.79.0'
```

Comment:
Rather than use the latest version (`0.80.0` now) which requires changes to the test assertions, I thought it better to keep the maintenance branches unchanged, and we will update only the `main` branch to use the latest version and adapt to the changes needed. Let me know if anyone disagrees with that direction.